defaults/main.yml

```
@@ -93,6 +97,21 @@ devture_traefik_config_entrypoint_web_secure_forwardedHeaders_trustedIPs: []
 # Also see: `devture_traefik_config_entrypoint_web_secure_forwardedHeaders_trustedIPs`
 devture_traefik_config_entrypoint_web_secure_forwardedHeaders_insecure: false
 
+# Controls whether the metrics entrypoint is enabled
+devture_traefik_config_entrypoint_metrics_enabled: "{{ devture_traefik_config_metrics_prometheus_enabled }}"
+devture_traefik_config_entrypoint_metrics_name: metrics
```

```suggestion
devture_traefik_config_entrypoint_metrics_name: "{{ devture_traefik_config_metrics_prometheus_entrypoint }}"
```

In another review comment, I've introduced a new `devture_traefik_config_metrics_prometheus_entrypoint` variable that configures the entrypoint name.

Here, I'm making this variable (`devture_traefik_config_entrypoint_metrics_name`) follow the other one (`devture_traefik_config_metrics_prometheus_entrypoint`).